theme/less/_header.less

```diff
@@ -0,0 +1,41 @@
+header
+{
+  margin: 25px 30px; 
```

Comment:
Meeting notes:
- Let's make the sidebar as whole, to prevent the side border.

Reply:
Having consistency with the design is more important. So yes, in this case it's fine.